packages/client/src/exchanges/protectedEndpointExchange.ts

```diff
@@ -27,7 +28,7 @@ const protectedEndpointExchange: Exchange =
         if (isPrivilegedLocation) {
           // we're updating the operation on the fly
           // eslint-disable-next-line no-param-reassign
-          op.context.url = protectedUrl;
+          op.context.url = protectedUrl ?? "";
```

Comment:
This null coalesce is unnessary because of the ternary above, right?

Reply:
Good catch, also thanks for the help on the dev server stuff. This was fixed as part of that fix.

[Fix dev server api](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10611/commits/3e10072ff0d2c83756686d7406143ce6b4c50a56)